WcaOnRails/db/migrate/20170320222511_change_default_entry_fee_for_competitions.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+class ChangeDefaultEntryFeeForCompetitions < ActiveRecord::Migration[5.0]
+  def change
+    Competition.where(base_entry_fee_lowest_denomination: nil).update_all(base_entry_fee_lowest_denomination: 0)
+    change_column_default(:Competitions, :base_entry_fee_lowest_denomination, from: nil, to: 0)
```

Comment:
Sorry to keep complaining about already reviewed code, but I don't see anything limiting fees to positive numbers. That could be done with a DB constraint, but I think it's better to have a Ruby validation. Unless there is some exotic use case for negative fees?